docker-compose.example.yml

```diff
@@ -13,14 +14,20 @@ services:
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock"
       - "/var/lib/docker/containers/:/var/lib/docker/containers/"
-      - "/var/lib/pterodactyl/:/var/lib/pterodactyl/"
+      - "/etc/pterodactyl/:/etc/pterodactyl/"
+      - "/var/log/pterodactyl/:/var/log/pterodactyl/"
```

Comment:
I feel like all of these pterodactyl specific mounts should live in a single root directory for the instance, rather than scattered all over the host filesystem. That's how I've seen many other docker setups work. e.g. `~/wings/[logs|lib|etc.]`. I feel like scattering them all over, especially given how many things get mounted, is counter to the point of docker.

Reply:
linux standard is to put them where they are now.

I could add symlinks in the container but I have seen those fail.

Reply:
Also the daemon expects some of these to be in specific places on the host as well due to the bind mounts foor the containers.

Reply:
@DaneEveritt Do you have a recommendation for where I "should" put these?

Reply:
Some type of Pterodactyl specific data directory. I mean, `/etc/pterodactyl/[etc,lib,log]` could work, or whatever the most logical place to put all of the stuff for a given program on the OS would be.

Reply:
We cant really consolidate these. I have the volumes and the reason why.

/var/run/docker.sock - Docker control
/var/lib/docker/containers/ - Docker logs
/etc/pterodactyl/ - wings config
/var/lib/pterodactyl/ - pterodactyl servers
/var/log/pterodactyl/ - pterodactyl logging (standard location, could probably move.)
/tmp/pterodactyl/ - installl scripts 

/srv/daemon-data/ - updated from old wings installl
/etc/letsencrypt/ - easier for letsencrypt automation for certs.

Reply:
```
 /etc/pterodactyl/ - wings config
/var/lib/pterodactyl/ - pterodactyl servers
/var/log/pterodactyl/ - pterodactyl logging (standard location, could probably move.)
/tmp/pterodactyl/ - installl scripts
```

All four of these can safely be mounted as `/etc/pterodactyl/[etc,lib,log,tmp]:[respective in-container path]`